ai-ml/jupyterhub/helm-values/jupyterhub-values-dummy-gpu.yaml

```diff
@@ -90,11 +90,11 @@ singleuser:
               display_name: "PyTorch 1.13.1 + torch-neuronx"
               default: true
               kubespawner_override:
-                image: 763104351884.dkr.ecr.us-west-2.amazonaws.com/pytorch-inference-neuronx:1.13.1-neuronx-py310-sdk2.13.2-ubuntu20.04
+                image: 054678736585.dkr.ecr.us-east-1.amazonaws.com/jupyterhub-inferentia-pytorch:1
```

Comment:
Lets obfuscate the account ids.

Reply:
Agree. Push these images to public ECR under `data-on-eks`.